### ISSUE TITLE
code comments match background color too closely so it is hard to read

### ISSUE THREAD

**Issue by RobKohr** [robb0wen/synthwave-vscode#124]
![image](https://user-images.githubusercontent.com/26756/71279778-9f357980-2327-11ea-9dd7-97ef0fad2830.png)

In comparison to the contrasting colors rest of the code, comments are nearly invisible, but comments really are the most important thing to see and read as it summarizes what the code is doing.

Can you please brighten them up a bit so they can stand out some. Just a little more brightness would go a long way.

**Comment by RobKohr** [comment]
I went as far as making it white in my settings:
```
"editor.tokenColorCustomizations":{
        "[SynthWave '84]": {"comments": "#FFFFFF"}
    },
```
![image](https://user-images.githubusercontent.com/26756/71280610-ac536800-2329-11ea-9d1a-2f1ba1ef3617.png)
which doesn't really work with this style, but stands out nicely.

**Comment by pangolingo** [comment]
Ditto on this. I find that the comments are too low-contrast against the background. When there's an active selection on the comments, they become impossible to read (unless you defocus the window—then the selection becomes more transparent.

screenshot of a selected comment:
![image](https://user-images.githubusercontent.com/1423865/71914300-f36d9580-313e-11ea-8627-eaa3fba4ea99.png)

**Comment by thericktastic** [comment]
I'm confused about this. I was just using VSCode with this theme on Friday, and it appeared as below in the screenshot. When I opened VSCode today it looks like your screenshots, Rob.

I'm not sure what happened, but I'm a much bigger fan of the aesthetic in my screenshot than how it looks now. 
<img width="1680" alt="screenshot" src="https://user-images.githubusercontent.com/54926729/74171795-14714c80-4bfd-11ea-9165-a3cd9e58f527.png">

**Comment by jwhitley** [comment]
Agreed. To add a little concrete info: the contrast between the background `#262335` and the comment foreground `#495495` is only 2.17:1.  For example, WCAG guidelines require a minimum contrast of 4.5 for accessibility reasons. The effective contrast will also worsen depending on the user's display brightness; generally harder to read the dimmer the display is set.

I'm evaluating bumping the contrast of the current comment color to hit 4.5. That produces a comment color of `#7f87c4`, using [HSLuv colorspace on Leonardo](https://leonardocolor.io/?colorKeys=%23495495&base=262334&ratios=3%2C4.5%2C5.5&mode=HSLuv). This seems a lot better in my early evaluation, including remaining readable even when the screen is a lot dimmer than local lighting conditions would suggest.

```
  "editor.tokenColorCustomizations": {
    "[SynthWave '84]": {
      "comments": "#7f87c4"
    }
  },
```

(An aside for folks who are new to Leonardo: the trick for this usage is to set "Base Color" to the background color, then click on the one key color swatch and set that to the problematic color. In this discussion, that would be the comments color or the selection color. The numbers in the left column are contrasts vs. the base color interpolated in the selected colorspace. IMO, HSLuv produced a nicer result relative to Synthwave84's design than some of the other options. The RGB values for the selected contrasts on the left appear in the right column.)

On @pangolingo's comment above, I'd recommend making the selection color a little darker if it's a still a problem after adjusting the comment color. You just need to be able to see the selected region against the background, so that can be a lower contrast than 4.5 and still work well for many users.

**Comment by miltoncandelero** [comment]
following @jwhitley and @pangolingo I landed on "#c2c5e3" to have a nice contrast with the selected text color (#545260)
https://leonardocolor.io/?colorKeys=%23495495&base=545260&ratios=3%2C4.5%2C5.5&mode=HSLuv

**Comment by moxwel** [comment]
I know this is a bit late, but I wanted to fix this; because yes, this is too low contrast, and it's hard to read, even more when that text is selected.

@jwhitley proposed `#7F87C4` for the **comment text**, and `#FFFFFF20` for **selection background**.

@miltoncandelero proposed `#C2C5E3` for the **comment text**.

Putting this two colors on the "new" selection background and normal background (plus the "normal" plaintext color) would be something like this:

![colors](https://user-images.githubusercontent.com/49621788/82708309-ee236a00-9c4b-11ea-9653-64d253af3b08.png)

I think `#C2C5E3` is way too similar to the normal plaintext color, and `#7F87C4` is still somewhat dark and low contrast. I think something like "in between" would be better, and I found `#949CD3`.

![colors2](https://user-images.githubusercontent.com/49621788/82711077-6d1ba100-9c52-11ea-9347-596c9feba105.png)

A comparison:

![comp](https://user-images.githubusercontent.com/49621788/82712737-bb32a380-9c56-11ea-9f38-3a709b57de83.png)

**Comment by robb0wen** [comment]
This is resolved in v0.1.7 thanks to @moxwel
